tools/cli/commands/docs.js

```diff
@@ -9,16 +9,12 @@ import path from 'path';
  */
 export function docsDefine( yargs ) {
 	yargs.command(
-		'docs [project]',
-		'Parses documentation from a project and outputs them into a JSON file.',
+		'docs [path]',
+		'Parses PHPDoc documentation from a project and outputs it into a JSON file.',
 		yarg => {
-			yarg.positional( 'project', {
-				describe:
-					'Project in the form of type/name, e.g. plugins/jetpack, ' +
-					'or type, e.g. plugins, or "all". Note that "all" means' +
-					'the Jetpack plugin plus all packages.',
+			yarg.positional( 'path', {
+				describe: 'Path to a jetpack-production checkout folder',
```

Comment:
Since we can run it on anything, may as well provide this as an example rather than implying it's only for Jetpack.

```suggestion
				describe: 'e.g. path to a jetpack-production folder',
```